### ISSUE TITLE
thickened interventricular septum 

### ISSUE THREAD

**Issue by gawlinski** [obophenotype/human-phenotype-ontology#3621]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Thickened interventricular septum 

### 2. Synonyms
 Hypertrophied interventricular septum, increased interventricular septum wall thickness

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Seip-Berardinelli Syndrome

**Comment by pnrobinson** [comment]
We need to revist this term,
https://hpo.jax.org/app/browse/term/HP:0005144
and provide a definition. I think it might be a synonym although we need to look at the annotated diseases.

**Comment by pnrobinson** [comment]
https://hpo.jax.org/app/browse/term/HP:0005144 should be renamed "Ventricular septal hypertrophy". The "left" doesn't make sense anyway as the septum is between left and right. This matches three currently annotated diseases. I think the reason that we had this is that sometimes a thickened interventricular septum can bulge into the left ventricle and cause outflow tract obstruction.

**Comment by pnrobinson** [comment]
Also consider if we need a term for ventricular septal bulge (VSB) see http://heart.bmj.com/content/102/14/1087.full

**Comment by pnrobinson** [comment]
The OMIM entry for Fabry disease is the only reference I can find to "Left" ventricular septal hypertrophy. The remaining HPO annotations are for "ventricular septal hypertrophy" I have asked the OMIM team to clarify but probably we can/should remove the "Left" from the term label and then this is a synonym of this NTR.

See also
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596758/#EHS166C8

**Comment by pnrobinson** [comment]
Joanna agreed that "left" was mistaken. I am revising the label.
